joint1_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: joint_1f
    joint_speed: 0.2
    motor:
        id: 6
        init: 0
        min: -2047
        max: 2047
        
joint2_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller_dual_motor
        type: JointPositionControllerDual
    joint_name: joint_2f
    joint_speed: 1.17
    motor_master:
        id: 2
        init: 0
        min: -4096
        max: 0
    motor_slave:
        id: 4

joint3_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: joint_3f
    joint_speed: 0.2
    motor:
        id: 3
        init: 0
        min: -2047
        max: 2047
